packages/local-workspace/src/remote_map/remote_map.ts

```diff
@@ -788,15 +788,19 @@ export const createRemoteMapCreator = (
             throw new Error('can not flush a non persistent remote map')
           }
 
+          log.debug('flushing %s', namespace, { wasClearCalled })
```

Comment:
format string does not match the extra args
I think we don't actually need `wasClearCalled` here, but if we do, it should be
```suggestion
          log.debug('flushing %s. %o', namespace, { wasClearCalled })
```
or, for better formatting
```suggestion
          log.debug('flushing %s, wasClearCalled=%s', namespace, wasClearCalled)
```